docs/security.md

```diff
@@ -186,7 +186,52 @@ configure those ports.
   </tr>
 </table>
 
+### HTTP Security Headers
+
+Apache Spark can be configured to include HTTP Headers which aids in preventing Cross 
+Site Scripting (XSS), Cross-Frame Scripting (XFS), MIME-Sniffing and also enforces HTTP 
+Strict Transport Security.
+
+<table class="table">
+    <tr><th>Property Name</th><th>Default</th><th>Meaning</th></tr>
+    <tr>
+        <td><code>spark.ui.xXssProtection</code></td>
+        <td>None</td>
+        <td>
+            Value for HTTP X-XSS-Protection response header. You can choose appropriate value 
+            from below:
+            <br />
```

Comment:
Why not just leave this as a bulleted list? Not a big deal I guess just less conventional for HTML

Reply:
@srowen Made them bulleted list and closed the <li> tag as well.